sapi/phpdbg/phpdbg.h

```diff
@@ -107,8 +108,6 @@
 #define memcpy(...) memcpy_tmp(__VA_ARGS__)
 #endif
 
-#if !defined(PHPDBG_WEBDATA_TRANSFER_H) && !defined(PHPDBG_WEBHELPER_H)
```

Comment:
Unrelated, but do we still need those memcpy defines above? Looks really weird.

Reply:
I have no idea, apparently it's related to apple ... @bwoebi ?